quarkus/tests/integration/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java

```diff
@@ -164,7 +162,6 @@ private void waitForReadiness() throws MalformedURLException {
                 connection.connect();
 
                 if (connection.getResponseCode() == 200) {
-                    LOGGER.infof("Keycloak is ready at %s", contextRoot);
```

Comment:
prevented to run all tests in IDE bc. this log entry was delayed for some reason and then in the logstream for tests where we compare log output..